src/scheduled_tasks.rs

```diff
@@ -654,4 +580,16 @@ mod tests {
     assert_eq!(form.software.ok_or(LemmyErrorType::NotFound)?, "mastodon");
     Ok(())
   }
+
+  #[tokio::test]
+  #[serial]
+  async fn test_scheduled_tasks_no_errors() -> LemmyResult<()> {
+    let context = test_context().await;
+
+    startup_jobs(&mut context.pool()).await?;
+    update_instance_software(&mut context.pool(), context.client()).await?;
+    delete_expired_captcha_answers(&mut context.pool()).await?;
+    publish_scheduled_posts(&context).await?;
+    Ok(())
+  }
```

Comment:
Strangely the test is passing even with the old, broken data in utils.sql. I even tried to add a user account and checked that the scheme update is executed, but everything seems fine and yet no errors is thrown. Any idea why?